minkindr_gtsam/test/test_minkindr-gtsam.cc

```diff
@@ -225,10 +219,27 @@ TEST(MinkindrGtsamTests, testSO3Log) {
   EQuaternion C(1);
   EVector3 logC = kindr::minimal::quaternionLog(C);
 
-  const double fd_step = 1e-9;
-  const double tolerance = 1e-6;
   SCOPED_TRACE("Testing Expression Jacobians.");
   testExpressionJacobians(logC, values, fd_step, tolerance);
+
+  for(int i = 0; i < N_TEST_ITERATIONS; ++i) {
+    Cval.setRandom();
+    values.update(1, Cval);
+    SCOPED_TRACE("Testing Expression Jacobians.");
+    testExpressionJacobians(logC, values, fd_step, tolerance);
+  }
+
+  // Testing at values near identity quaternion
+  Eigen::Vector3d aa(0,0,0);
+  for(int i = 0; i < N_TEST_ITERATIONS; ++i) {
+    Cval = Quaternion(aa);
+    values.update(1, Cval);
```

Comment:
Ah that is the test I was missing  ... sorry should have read till the end. I'm fine then.